src/wristpy/processing/analytics.py

```diff
@@ -225,25 +225,44 @@ def _find_long_blocks(
     ) -> np.ndarray:
         """Helper function to find long blocks where SPT window is true.
 
-        This function uses the convolution of a kernel of 1s, of length block_length,
-        to find the continuous long blocks where SPT window is true. Where the
-        convolution value is == long_block_length that implies an overlap between the
-        kernel and the threshold signal of length long_block.
+        This function finds the first non-zero in below_threshold array, if there are
```

Comment:
This is definitely going into documentation overkill territory -- let the code speak for itself more.